StripeFinancialConnections/StripeFinancialConnections/Source/Native/NetworkingLinkVerification/NetworkingLinkVerificationViewController.swift

```diff
@@ -0,0 +1,192 @@
+
+        showLoadingView(true)
+        dataSource.startVerificationSession()
+            .observe { [weak self] result in
+                guard let self = self else { return }
+                self.showLoadingView(false)
+                switch result {
+                case .success(let consumerSessionResponse):
+                    self.showContent(redactedPhoneNumber: consumerSessionResponse.consumerSession.redactedPhoneNumber)
+                case .failure(let error):
+                    self.dataSource.analyticsClient.log(
+                        eventName: "networking.verification.error",
+                        parameters: [
+                            // TODO(kgaidis): figure out a proper way to log this error
+                            "error": "here"
+                        ],
+                        pane: .networkingLinkVerification
+                    )
+                    self.delegate?.networkingLinkVerificationViewController(self, didReceiveTerminalError: error)
+                }
+            }
+    }
+
+    private func showContent(redactedPhoneNumber: String) {
+        let pane = PaneWithHeaderLayoutView(
+            title: STPLocalizedString(
+                "Sign in to Link",
+                "The title of a screen where users are informed that they can sign-in-to Link."
+            ),
+            subtitle: STPLocalizedString(
+                "Enter the code sent to \(redactedPhoneNumber)",
+                "The subtitle/description of a screen where users are informed that they have received a One-Type-Password (OTP) to their phone."
+            ),
+            contentView: bodyView,
+            footerView: nil
+        )
+        pane.addTo(view: view)
+    }
+
+    private func showLoadingView(_ show: Bool) {
+        if show && loadingView.superview == nil {
+            // first-time we are showing this, so add the view to hierarchy
+            view.addAndPinSubview(loadingView)
+        }
+
+        loadingView.isHidden = !show
+        if show {
+            loadingView.startAnimating()
+        } else {
+            loadingView.stopAnimating()
+        }
+        view.bringSubviewToFront(loadingView)  // defensive programming to avoid loadingView being hiddden
+    }
+}
+
+// MARK: - NetworkingLinkVerificationBodyViewDelegate
+
+@available(iOSApplicationExtension, unavailable)
+extension NetworkingLinkVerificationViewController: NetworkingLinkVerificationBodyViewDelegate {
+
+    func networkingLinkVerificationBodyView(
+        _ view: NetworkingLinkVerificationBodyView,
+        didEnterValidOTPCode otpCode: String
+    ) {
+        view.otpTextField.text = "CONFIRMING OTP..."
+
+        dataSource.confirmVerificationSession(otpCode: otpCode)
+            .observe { [weak self] result in
+                guard let self = self else { return }
+                switch result {
+                case .success:
+                    view.otpTextField.text = "SUCCESS! CALLING markLinkVerified..."
```

Comment:
these are debug/interim code right?

Reply:
@vardges-stripe yes, it will be later replaced by a nicer OTP UI component